docs/bigcommerce-for-wordpress/extending-the-plugin/customization-guide.md

```diff
@@ -257,12 +278,87 @@ remove_action( 'bigcommerce/template/product/archive', bigcommerce()->templates-
 add_action( 'bigcommerce/template/product/archive', 'your_callback_function', 10, 2 );
 ```
 
+## Localizing strings displayed on the storefront
+
+The language used by BC4WP to display messages on your storefront can be localized. The array `$js_i18n_array` defined in `src/BigCommerce/Assets/Theme/JS_Localization.php:19` contains all strings used by `assets/js/dist/scripts.js` to display messages on the front-end.
+
+```php
+$js_i18n_array = [
+			'operations' => [
+				'query_string_separator' => __( '&', 'bigcommerce' ),
+			],
+			'cart'       => [
+				'items_url_param'          => '/items/',
+				'mini_url_param'           => '/mini/',
+				'quantity_param'           => 'quantity',
+				'message_empty'            => __( 'Your cart is empty.', 'bigcommerce' ),
+				'continue_shopping_label'  => esc_html( $empty_cart_data[ Cart_Empty::LINK_TEXT ] ),
+				'continue_shopping_url'    => esc_url( $empty_cart_data[ Cart_Empty::LINK ] ),
+				'cart_error_502'           => __( 'There was an error with your request. Please try again.', 'bigcommerce' ),
+				'add_to_cart_error_502'    => __( 'There was an error adding this product to your cart. It might be out of stock or unavailable.', 'bigcommerce' ),
+				'ajax_add_to_cart_error'   => __( 'There was an error adding this product to your cart.', 'bigcommerce' ),
+				'ajax_add_to_cart_success' => __( 'Product successfully added to your cart.', 'bigcommerce' ),
+				'mini_cart_loading'        => __( 'Loading', 'bigcommerce' ),
+				'shipping_calc_error'      => __( 'There was an error calculating your shipping cost. Please try again.', 'bigcommerce' ),
+			],
+			'account'    => [
+				'confirm_delete_message' => __( 'Are you sure you want to delete this address?', 'bigcommerce' ),
+				'confirm_delete_address' => __( 'Confirm', 'bigcommerce' ),
+				'cancel_delete_address'  => __( 'Cancel', 'bigcommerce' ),
+			],
+			'errors'     => [
+				'pagination_error'         => __( 'There was an error processing your request. Please try again.', 'bigcommerce' ),
+				'pagination_timeout_error' => __( 'The server took too long to complete this request. Please try again.', 'bigcommerce' ),
+			],
+			'pricing'    => [
+				'loading_prices' => __( 'Retrieving current pricing data...', 'bigcommerce' ),
+			],
+			'inventory'    => [
+ 				'in_stock' => __( 'in Stock', 'bigcommerce' ),
+ 				'out_of_stock' => __( 'Out of Stock', 'bigcommerce' ),
+			],
+			'wish_lists' => [
+				'copied' => __( 'Copied!', 'bigcommerce' ),
+				'copy_link' => __( 'Copy link', 'bigcommerce' ),
+				'copy_success' => __( 'Wish List URL copied to clipboard.', 'bigcommerce' ),
+			],
+		];
+```
+
+As an example of how to localize strings, let's change the message displayed to a user when they add an item to their cart. By default, the message "Product successfully added to your cart." is displayed. Instead, let's update that language and include a link to your store's cart.

```

Comment:
As an example of how to localize strings, let's change the message displayed to a user when they add an item to their cart. By default, the message "Product successfully added to your cart."  displays. Instead, let's update that language and include a link to your store's cart.